public/individualdevstats.php

```diff
@@ -351,7 +351,7 @@
         $firstAchievement = $achievement;
         $lastAchievement = $achievement;
     } else {
-        if ($hardestAchievement['Points'] && $achievement['Points'] && ($hardestAchievement['TrueRatio'] / $hardestAchievement['Points']) < ($achievement['TrueRatio'] / $achievement['Points'])) {
+        if ($hardestAchievement['Points'] == 0 || $hardestAchievement['Points'] && $achievement['Points'] && ($hardestAchievement['TrueRatio'] / $hardestAchievement['Points']) < ($achievement['TrueRatio'] / $achievement['Points'])) {
```

Comment:
I think this would be clearer if the old logic was in a set of parenthesis, possibly on a separate lin:
```
if ($hardestAchievement['Points'] == 0 || 
    ($hardestAchievement['Points'] && $achievement['Points'] && ($hardestAchievement['TrueRatio'] / $hardestAchievement['Points']) < ($achievement['TrueRatio'] / $achievement['Points']))) {
```
For performance reasons, you could also calculate `$hardestAchievement['TrueRatio']/$hardestAchievement['Points']` outside the loop and put the new logic there.

Reply:
Nevermind, that won't work. As `hardestAchievement` could change within the loop.

Reply:
Good point, this was mainly just a hotfix that I pulled out from the other changes I'm making in individualdevstats.php which aren't ready yet.
I can make the formatting suggestion along with the `$easiestAchievement  == 0 ||` version below it in my next PR for this (already added them to it) if that is fine. Otherwise I'll add it here if that is preferred.

Reply:
Fair enough.